core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/CommonLoadBalancer.scala

```diff
@@ -283,7 +288,7 @@ abstract class CommonLoadBalancer(config: WhiskConfig,
         totalActivationMemory.add(entry.memoryLimit.toMB * (-1))
         activationsPerNamespace.get(entry.namespaceId).foreach(_.decrement())
 
-        releaseInvoker(invoker, entry)
+        invoker.foreach(releaseInvoker(_, entry))
```

Comment:
👍 

If this is to use some methods of `CommonLoadBalancer` in the downstream, isn't there any other methods to be updated as well?